core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -189,6 +189,10 @@ public Slave getNode() {
         }
     }
 
+    protected TaskListener gettaskListener() {
```

Comment:
getTaskListener() . Actually I would prefer a more explicit method name (e.g. getSlaveLog() or whatever). Could be also fixed by Javadoc.